setup.py

```diff
@@ -16,29 +16,36 @@
 #   TORCH_XLA_PACKAGE_NAME
 #     change the package name to something other than 'torch_xla'
 #
-#   COMPILE_PARALLEL=1
-#     enable parallel compile
-#
-#   BUILD_CPP_TESTS=1
-#     build the C++ tests
-#
-#   XLA_DEBUG=0
-#     build the xla/xrt client in debug mode
-#
-#   XLA_BAZEL_VERBOSE=0
+#   BAZEL_VERBOSE=0
 #     turn on verbose messages during the bazel build of the xla/xrt client
 #
 #   XLA_CUDA=0
 #     build the xla/xrt client with CUDA enabled
 #
+#   XLA_CPU_USE_ACL=0
+#     whether to use ACL
```

Comment:
can you explain more about what this access control list does for xla_cpu?

Reply:
I have no idea what this is :)

This is the current behaviour and it comes from here:
https://github.com/pytorch/xla/blob/HEAD/build_torch_xla_libs.sh#L50

I attempted to replicate it and not change semantics.
Probably someone from the team can answer this.

Reply:
it is from AWS CPU team, they want to use pytorch/xla for ARM CPU I think.